train.py

```diff
@@ -75,22 +51,24 @@ def save_model(model, epoch):
 	}
 	torch.save(model_state, filename)
 
-def train(num_epochs, model, train_loader,test_loader, optimizer, criterion, device,log_interval):
+def train(num_epochs, model, train_loader,test_loader, optimizer, criterion, device,log_interval,vocab):
 	"""
 	Executes model training. Saves model to a file every epoch.
 	"""	
 
 	for epoch in range(1,num_epochs+1):
-
-		train_single_epoch(epoch, model, train_loader,optimizer, criterion, device, log_interval)
+		epoch_start_time=time.time()
+		train_single_epoch(epoch=epoch,model=model,train_loader=train_loader,optimizer=optimizer, criterion=criterion,device= device,log_interval=log_interval)
 
 		if epoch % 5 == 0:
```

Comment:
we should use the `log_interval` variable here right?

Reply:
Same as for the other comment